### PR TITLE
Feature/Optional Upload Report

### PR DESCRIPTION
**Pull request by kdavis**
# Why
Added a new property to optionally upload a test report - sometimes uploading an artifact isn't always needed (and potentially useful to skip)

# What
- Added new table entry in documentation
- Added new clause to check against the new config property

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ This GitHub Action automates the execution of GdUnit4 unit tests within the Godo
 | arguments      | Additional arguments to pass to GdUnit4<br> see https://mikeschulze.github.io/gdUnit4/advanced_testing/cmd/. | string | false    |           |
 | timeout        | The test execution timeout in minutes.                  | int    | false    | 10        |
 | retries        | The number of retries if the tests fail.                | int    | false    | 0         |
+| upload-report | Whether to publish & upload the report file | bool    | false    | true      |
```
Comment (MikeSchulze) [inline README.md:39]: Please use the same formattings for the table

File: README.md [hunk 2]
```diff
@@ -73,9 +74,9 @@ A GdUnit4 **version** should be specified as a string, such as `v4.2.1`. To run
     timeout: ''
 
     # The number of retries if the tests fail. (This can be used for flaky test)
-    # Default: 0 
+    # Default: 0
     retries: ''
-    
+
```
Comment (MikeSchulze) [inline README.md:79]: And add here the default example like the others.

**Review by MikeSchulze — CHANGES_REQUESTED** [review]
Hi, at first, thanks for contribution to this project.

Only minor things, but looks good so far.

Please create a feature request issue and describe what and why you do want this.
Then attach the issue number to the PR.

An Issue is tracked by creating the release nodes and will be listed in the release nodes.

Thank you ;)

**Comment by kdavis** [comment]
Raised feature enhancement request #23 :D

**Review by MikeSchulze — APPROVED** [review]
👍

**Pull request merged**